content/courses/Ada_For_The_C_Embedded_Developer/chapters/06_Translation.rst

```diff
@@ -1167,7 +1167,7 @@ shifting and masking to access the bits of that byte. Here, we use the
 
     void transmit (void *bits, int len)
     {
-        int i, j;
+        int i;
```

Comment:
We're actually using ``j`` below...

Reply:
Investigating :) I think I may have made this change to the wrong code block.